certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -4880,6 +4880,148 @@
         }
       }
     },
+    "/v2/FlowV2/ClassifyDocument": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "Classify or Identify documents based on the file content. Please refer here for more info - https://dev.pdf4me.com/power-automate/documentation/actions/classify-document/",
+        "operationId": "ClassifyDocument_V1",
```

Comment:
@varunynoox operationIds should be in Camel Case. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please change to "ClassifyDocumentV1".

Reply:
operaitonid has been renamed